cli/lib/kontena_cli.rb

```diff
@@ -87,6 +115,26 @@ def self.cli_root(*joinables)
       File.join(Kontena.root, 'lib/kontena/cli', *joinables)
     end
   end
+
+  def self.logger
+    return @logger if @logger
+    if log_target.respond_to?(:tty?) && log_target.tty?
+      logger = Logger.new(log_target)
+      logger.datetime_format = "%M%S.%2N "
```

Comment:
I suggest something like `%T.%3N`. It's short and makes sense when you first see it.

```
irb(main):001:0> Time.now.strftime('%T.%3N')
=> "12:19:22.053"
```